packages/zoe/src/contracts/vpool-xyk-amm/priceAuthority.js

```diff
@@ -5,7 +5,7 @@ import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import { observeNotifier } from '@agoric/notifier';
 
-import { makeOnewayPriceAuthorityKit } from '../../contractSupport/index.js';
+import { makeOnewayPriceAuthorityKit } from '../../contractSupport';
```

Comment:
Was this change on purpose? I think bundleSource won't like it.

Reply:
Not on purpose. Must have been done automatically when moving the file.  I think prettier is always suggesting shortening names like this, and I never manually accept, but I'll bet it can automatically fix it when I'm not looking in some cases.